src/main/java/org/opentripplanner/model/GroupOfStations.java

```diff
@@ -28,20 +29,17 @@ public GroupOfStations(FeedScopedId id) {
     }
 
 
-    public String getName() {
+    public I18NString getName() {
```

Comment:
I tried to check if this was in use(e.g. reflection or cast) in the APIs where this change would be harmful, but it looks like it is not in use.